.github/workflows/wf-edgex-cli.yml

```diff
@@ -0,0 +1,69 @@
+name: EdgeXFoundry-Snap
+on:
+  schedule:
+  - cron:  "0 6 * * *"
+jobs:
+  sync-repo:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Sync and version repo
+        uses: canonical/edgex-sync-and-create-launchpad-branch-action@v1.4.0
+        with:
+          edgex-repo: "edgex-go"
+          canonical-repo: "edgex-go"
```

Comment:
This was my mistake - I left this from the instructions and will update the CI document. These two lines - "edgex-repo" and "canonical-repo" should be set to the name of the upstream and canonical repositories. in both cases they should be "edgex-cli" instead of "edgex-go"